libmachine/drivers/plugin/register_driver.go

```diff
@@ -29,8 +28,6 @@ Please use this plugin through the main 'docker-machine' binary.
 		os.Exit(1)
 	}
 
-	libmachine.SetDebug(true)
```

Comment:
Drivers no longer need to set debug - all logs are sent back to the main process, and are forwarded (or not) depending on the overall log level.